src/components/MinorList/index.js

```diff
@@ -13,14 +13,28 @@ export default class Minors extends Component {
   render() {
     let content;
     let minorPrefix;
+    // Gets the row item widths
+    let rowWidths = this.props.rowWidths.twoItems;
+    let rowItemOne = rowWidths.itemOne;
+    let rowItemTwo = rowWidths.itemTwo;
+
+    // Creates the list item's content
     if (this.props.minors) {
-      content = this.props.minors.map(minor => (
-        <div>
-          <Typography>{minor}</Typography>
-        </div>
-      ));
+      // Gets the last item of the list
+      let lastItem = this.props.minors[this.props.minors.length - 1];
+      // If there are multiple minors, each minor name will have a comma after it except for the
+      // very last. If there's only one minor, no comma will appear
+      let minorText = '';
+
+      this.props.minors.forEach(minor => {
+        // NOTE: The difference between the two statements that adds text to the variable
+        // "majorText" is that one of them adds BOTH a comma and a space
+        minor === lastItem ? (minorText += `${minor}`) : (minorText += `${minor}, `);
+      });
+      content = <Typography>{minorText}</Typography>;
```

Comment:
Updates the way a student's minors are displayed. Before, if the student had multiple minors, their minor names would appear in a column. This creates a lot of white space due to multiple lines being created. This has been changed so that the minor names are on the same line and are separated by a comma. If there's only one minor, no comma will display.